File: dmr-api/Models/PeriodMixing.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace DMR_API.Models
{
    public class PeriodMixing
    {
        public int ID { get; set; }
        public int BuildingID { get; set; }
        public bool IsOvertime { get; set; }

        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }

        public DateTime CreatedTime { get; set; }
        public DateTime? UpdatedTime { get; set; }
        public DateTime? DeletedTime { get; set; }

        public bool IsDelete { get; set; }
        public int CreatedBy { get; set; }
        public int DeletedBy { get; set; }
        public int UpdatedBy { get; set; }
        public List<PeriodDispatch> PeriodDispatchList { get; set; }

    }
}

File: dmr-app/src/app/_core/_model/period.mixing.ts
import { PeriodDispatch } from "./period.dispatch";

export interface PeriodMixing {
    id: number;
    buildingID: number;
    isOvertime: boolean;
    startTime: any;
    endTime: any;
    createdTime: Date;
    updatedTime: Date | null;
    deletedTime: Date | null;
    isDelete: number;
    createdBy: number;
    deletedBy: number;
    updatedBy: number;
    periodDispatchList: PeriodDispatch[];
}
